wunderio/next-drupal-starterkit
# Update npm dependencies

## Link to feature environment:

[Drupal](https://feature-npm-updates.next-drupal-starterkit.dev.wdr.io)
[Next.js](https://feature-npm-updates-next.next-drupal-starterkit.dev.wdr.io/)
[Storybook](https://feature-npm-updates-storybook.next-drupal-starterkit.dev.wdr.io)

## Changes proposed in this PR:

- Note that this branches from https://github.com/wunderio/next-drupal-starterkit/pull/226, so most commits you see here are from that branch until that has been merged to main. Only e4ed013ea8c9b7b914f3a11cab394c28c384829c is for this PR.
- Update npm dependencies to latest. The [commit message](e4ed013ea8c9b7b914f3a11cab394c28c384829c) has more details about how this was done, which weren't updated and why, etc. 

Before: `4 high severity vulnerabilities`
After: `found 0 vulnerabilities`

## How to test:

1. Run the usual ./setup.sh -c locally, should be no errors
2. Check CircleCI build logs have nothing unexpected
3. Check feature environments work nicely
4. You can do more local testing if you wish; see commit message for the commands I tested with locally.
